third_party/terraform/tests/resource_cloud_scheduler_job_test.go

```diff
@@ -0,0 +1,204 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	cloudscheduler "google.golang.org/api/cloudscheduler/v1beta1"
```

Comment:
If you are going to use this client, cloudscheduler needs to be vendored directly through TF.

Reply:
I've removed this as it's not needed for the test